passes/techmap/abc.cc

```diff
@@ -1187,7 +1249,7 @@ void abc_module(RTLIL::Design *design, RTLIL::Module *current_module, std::strin
 
 			if (c->type.in(ID(_const0_), ID(_const1_))) {
 				RTLIL::SigSig conn;
-				conn.first = RTLIL::SigSpec(module->wires_[remap_name(c->connections().begin()->second.as_wire()->name)]);
+				conn.first = RTLIL::SigSpec(module->wire(remap_name(c->connections().begin()->second.as_wire()->name)));
```

Comment:
```suggestion
				conn.first = module->wire(remap_name(c->connections().front().as_wire()->name));
```

Reply:
`dict` has no `front()` member... But I removed the explicit `RTLIL::SigSpec` construction.